templates/zerver/help/unread-counter-and-pointer.md

```diff
@@ -0,0 +1,44 @@
+# How the unread counter and navigation work
+
+## Unread messages
+
+Zulip will consider a message read when the message is selected (the
+blue cursor box passes over it). Any message which is selected or
+above a message which is selected is marked as read. If you reach the
+very bottom of your feed, all the messages in view will be marked as
+read.
+
+## Navigation
+
+Zulip will always take you to the place you left off (your first
+unread message) to make it more easy for you to catch up with all the
+discussions that happened while you were away from your computer.
+
+Here is a more detailed overview of the navigation behavior:
+
+* When you click on a topic's name or the recipient list at the top of
+* a group of messages, Zulip will narrow you to that conversation and
+* Zulip will select the message you were previously focused on in that
+* conversation.
+
+* If you narrow into a conversation by using the left sidebar or the
+* search box, Zulip will instead select the first unread message
+* matching that narrow, or if there are none, the most recent messages
+* matching that narrow.
+
+* When you unnarrow you will automatically be taken to the same
+* message that was selected in the home view before you narrowed
+* (unless in the narrow you read new messages, in which case you will
+* be jumped forward to the first unread message).
+
+* When you open a new browser window or tab to the home view, Zulip
+* will select the furthest down that your cursor has ever reached in
+* the home view: this is usually just before the first unread message
+* in the home view.
+
+* When you load a new browser tab or window to a narrowed view, it's
+* as if you had loaded the browser window to your home view and then
+* clicked on the nearest message matching your narrow.
+
+You can find a more extensive explanation in our
+[developer documentation](http://zulip.readthedocs.io/en/latest/pointer.html).
```

Comment:
More generally. it might be helpful to port the style of this document away from the readthedocs/developer docs style towards a more `/help/` user docs style by, for example, adding in the tip and warning macros and including a screenshot or two.